plugins/node.d/snmp__winmem

```diff
@@ -86,6 +87,10 @@ my $total = $session->get_single('1.3.6.1.2.1.25.2.2.0') * 1024;
 if (defined $ARGV[0] and $ARGV[0] eq "config") {
 	my ($host) = Munin::Plugin::SNMP->config_session();
 
+my ($warning, $critical) = get_thresholds("used");
```

Comment:
Please take care to use the same indentation as the context.

Personally I would also prefer these initial lines (warning/critical) to be moved down to where they are used/printed. But this may be a matter of taste.